packages/opds-web-client/src/DataFetcher.ts

```diff
@@ -37,10 +37,6 @@ export default class DataFetcher {
     this.proxyUrl = config.proxyUrl;
     this.adapter = config.adapter;
     this.authKey = "authCredentials";
-
-    if (!this.adapter) {
```

Comment:
@aslagle removing this because the app re-renders and runs this code many times. Then this gets outputted in the console many times (falsely) until the adapter is set. I think there's a larger problem with how some parts of the app re-render more times than it should.

Reply:
Yeah, if it was up to me I would definitely prioritize performance and updating the app to follow react/redux best practices over all the UX stuff.